charts/edc-dataplane/Chart.yaml

```diff
@@ -3,8 +3,8 @@ apiVersion: v2
 name: edc-dataplane
 description: >-
   EDC Data-Plane - The Eclipse DataSpaceConnector data layer with responsibility of transferring and receiving data streams
-home: https://github.com/eclipse-tractusx/tractusx-edc
+home: https://github.com/catenax-ng/product-edc/charts/edc-dataplane
```

Comment:
Do not use NG organization in Eclipse-Tractusx. The home of the source code is the Eclipse-Tractusx repository, not the repo in the NG org.

Please add 
- License header to (all) yaml files
- key `source` to this file (refer to [TRG 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-2#the-chartyaml-file))